arch/x86/gen-syscall.bash

```diff
@@ -0,0 +1,130 @@
+#!/bin/bash
```

Comment:
I'm not sure I love bash as the tool here, but hey, I guess it's better than pulling a templating engine into the bst build!